### ISSUE TITLE
Request to create admin

### ISSUE THREAD

**Issue by kazashim** [eddycodes/Zambia-PAYE-Calculator#1]
i would like to improve interface

**Comment by eddycodes** [comment]
Nice, what ideas do you have in mind?

**Comment by kazashim** [comment]
> Nice, what ideas do you have in mind?

Well firstgiveit a nice admin look. then start adding some new features. e.g users

**Comment by eddycodes** [comment]
I'm down with that. Most of my work is in Laravel. I can see from your profile that you're also a fan. It would be fun turning this into a beast

**Comment by kazashim** [comment]
> I'm down with that. Most of my work is in Laravel. I can see from your profile that you're also a fan. It would be fun turning this into a beast

Yeah big time. am gonna setup a new Laravel project for it today i will upload the resources we will use

**Comment by kazashim** [comment]
here are some front end resource [https://drive.google.com/file/d/1J50gxWDJ6PNAXvJzWWO_mF4G5V6-5Mk5/view?usp=sharing](url)

**Comment by eddycodes** [comment]
Thanks for sharing. Drop your phone number and email so we can formally discuss the project

**Comment by kazashim** [comment]
phone: 0972150298
kazashimkuzasuwat@gmail.com
